src/SourceBuild/content/repo-projects/known-good.proj

```diff
@@ -34,6 +34,7 @@
         <!-- Tier 1 -->
         <RepositoryReference Include="command-line-api" />
         <RepositoryReference Include="diagnostics" />
+        <RepositoryReference Include="emsdk" />
```

Comment:
Feels like [Installer](https://github.com/dotnet/installer/blob/main/src/SourceBuild/content/repo-projects/installer.proj#L63) should have a dependency declared on emsdk.